src/consts.ts

```diff
@@ -1,3 +1,11 @@
+enum Page {
```

Comment:
i put this here since it seemed like a reasonable shared file to put it in, but please let me know if there's a better place! i'm new to the codebase